### PR TITLE
Fix a few spelling errors.

### PR DESCRIPTION
**Pull request by dzearing**
Just docsite updates.

### CODE REVIEW

File: docsite/docs/introduction.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ terminal, and fits nicely into CI loops and package script blocks.
 
 The CLI is controlled by command-line parameters and
 [package configuration](https://github.com/microsoft/rnx-kit/tree/main/packages/config).
-Command-line parameters are optional, but always take predecedence. Think of
+Command-line parameters are optional, but always take precedence. Think of
 them as overrides. Package configuration is _also_ optional, though it is
```
Comment (dzearing) [inline docsite/docs/introduction.md:47]:
```suggestion
Command-line parameters always override default values. Package configuration is optional but
```

Comment (afoxman) [reply docsite/docs/introduction.md:47]: This sounds good to me. I've pushed the button.

**Comment by afoxman** [comment]
Can you run `yarn change --empty` from the root and then `git push`? That is what is holding up your PR.

**Review by JasonVMo — APPROVED** [review]

**Comment by JasonVMo** [comment]
Approved (once the CI loop is fixed). Thanks for the fixes.

**Pull request merged**